mcproto/types/nbt.py

```diff
@@ -3,10 +3,10 @@
 from abc import abstractmethod
 from collections.abc import Iterator, Mapping, Sequence
 from enum import IntEnum
-from typing import ClassVar, Protocol, Union, cast, final, runtime_checkable
+from typing import ClassVar, Hashable, Protocol, Union, cast, final, runtime_checkable
```

Comment:
Huh, I didn't even know `typing` was re-exporting `collections.abc.Hashable`, it's not a generic type and it is available in 3.8, so there's no reason not to use the proper `collections.abc` version here.